sites/platform/src/add-services/opensearch.md

```diff
@@ -220,8 +263,8 @@ There are two ways to do so.
 
 ### Destructive
 
-In your `{{< vendor/configfile "services" >}}` file, change the version *and* name of your OpenSearch service.
-Then update the name in the `{{< vendor/configfile "app" >}}` relationships block.
+In your `{{< vendor/configfile "services" >}}` file, change the version *and* name of your Elasticsearch service.
```

Comment:
```suggestion
In your `{{< vendor/configfile "services" >}}` file, change the version *and* name of your Opensearch or Elasticsearch service.
```

Shouldn't we have both here? The intro of the **Upgrading** section implies that such upgrades can be needed when upgrading from one OS version to another, and not just from ES to OS?

Reply:
I think introducing Elasticsearch here was a copy-paste mistake on my part. Fixing.